forms-flow-documents/src/formsflow_documents/filters/custom_filters.py

```diff
@@ -0,0 +1,15 @@
+"""Custom filters module for jinja template."""
+
+
+def is_b64image(value: str) -> bool:
+    """Checks if the given string is base64 image."""
+    if not isinstance(value, str):
+        return False
+    is_image = False
+
+    if "data:image" not in value:
+        if "iVBORw0KGg" in value or "/9j/4" in value:
```

Comment:
Can't we simplify this check using `str in tuple` ?

Reply:
👍